tests/test_util.py

```diff
@@ -402,25 +421,38 @@ def request_callback(request):
 
                 digest = 'v1-converted-digest'
 
-        headers = {
-            'Content-Type': '{}+jsonish'.format(media_type_prefix),
-            'Docker-Content-Digest': digest
-        }
+        headers = {}
+        if digest_is_v1 or has_content_type_header:
```

Comment:
The digest_is_v1 check here is extremely hard to understand without a comment - my only guess is that some external component (pulp?) has the missing header behavior for v2 digests not v1 digests? If there's some parameterized combinations you want to skip, wouldn't it be better to bypass the whole test at the top?